client/settings/general-settings/test/general-settings.test.js

```diff
@@ -50,27 +57,49 @@ describe( 'GeneralSettings', () => {
 		}
 	);
 
-	it.each( [ [ true ], [ false ] ] )(
```

Comment:
A confirmation modal is displayed, so I needed to refactor this test a bit. With the refactor I made, having the `it.each` wouldn't have made things easier to read.